src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -830,14 +872,25 @@ static void Test3()
     }
 }
 ";
-            CreateCompilationWithMscorlibAndSpan(text).VerifyDiagnostics(
+            CreateCompilationWithMscorlibAndSpan(text, parseOptions: TestOptions.Regular10).VerifyDiagnostics(
```

Comment:
Will resume review here on Monday. Apologies that I didn't have the chance to get further. Have just skimmed the implementation and been working through the tests--then will go back to the implementation and apply more scrutiny.